etc/daemons/pollerd.ini

```diff
@@ -24,14 +24,14 @@ daemon_enabled=1
 #-- SSL configuration --
 use_ssl=0
 # WARNING : Put full paths for certs
-#ca_cert=/etc/alignak/certs/ca.pem
-#server_cert=/etc/alignak/certs/server.cert
-#server_key=/etc/alignak/certs/server.key
+#ca_cert=/usr/local/etc/alignak/certs/ca.pem
+#server_cert=/usr/local/etc/alignak/certs/server.cert
+#server_key=/usr/local/etc/alignak/certs/server.key
 #hard_ssl_name_check=0
 
 #-- Local log management --
 # Enabled by default to ease troubleshooting
 use_local_log=1
-local_log=%(logdir)s/pollerd.log
+local_log=/usr/local/var/log/alignak/pollerd.log
```

Comment:
This line is the example of the usage. logdir contains the value previously declared in the ini file. I don't get it.